go/README.md

```diff
@@ -0,0 +1,30 @@
+# Agent Config Go
+
+Agent config holds all the configuration settings for the Hypertrace Go Agent.
+
+## Getting Started
+
+```go
+// loads the config from the config file tbd.json and env vars
+cfg := config.Load()
```

Comment:
should load take an optional filename?

Reply:
Not a bad idea actually. Let me scratch it.